lib/open_exchange_rates/configuration.rb

```diff
@@ -1,5 +1,25 @@
 module OpenExchangeRates
   class Configuration
     attr_accessor :app_id
+
+    def cache
+      @cache ||= CacheConfiguration.new
+    end
+
+    def cache=(c)
+      @cache = c
+    end
   end
-end
\ No newline at end of file
+
+  class CacheConfiguration
+    attr_accessor :type, :client
+
+    def type
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.